server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/client/CloudFoundryConnection.java

```diff
@@ -0,0 +1,69 @@
+package org.opentosca.toscana.plugins.cloudfoundry.client;
+
+import java.util.List;
+
+import org.cloudfoundry.operations.CloudFoundryOperations;
+import org.cloudfoundry.operations.DefaultCloudFoundryOperations;
+import org.cloudfoundry.operations.services.ListServiceOfferingsRequest;
+import org.cloudfoundry.operations.services.ServiceOffering;
+import org.cloudfoundry.reactor.DefaultConnectionContext;
+import org.cloudfoundry.reactor.TokenProvider;
+import org.cloudfoundry.reactor.client.ReactorCloudFoundryClient;
+import org.cloudfoundry.reactor.tokenprovider.PasswordGrantTokenProvider;
+
+/**
+ implements java-cf-client
+ create a connection to the cf provider
+ */
+public class CloudFoundryConnection {
+
+    private String userName;
+    private String password;
+    private String apiHost;
+    private String organization;
+    private String space;
+    private CloudFoundryOperations cloudFoundryOperations;
+
+    public CloudFoundryConnection(String username, String password,
+                                  String apiHost, String organization,
+                                  String space) {
+
+        this.userName = username;
+        this.password = password;
+        this.apiHost = apiHost;
+        this.organization = organization;
+        this.space = space;
+
+        this.cloudFoundryOperations = createCloudFoundryOperations();
+    }
+
+    private CloudFoundryOperations createCloudFoundryOperations() {
+        DefaultConnectionContext connectionContext = DefaultConnectionContext.builder()
+            .apiHost(apiHost) //"api.run.pivotal.io"
+            .build();
+
+        TokenProvider tokenProvider = PasswordGrantTokenProvider.builder()
+            .password(password) //secret
+            .username(userName) //jmuell.dev@gmail.com
```

Comment:
Remove your credentials comments here, they dont belong in the code. (also applies for the rest of the similar comments in this file